app/src/main/java/com/example/sharingang/payment/StripePaymentProvider.kt

```diff
@@ -39,12 +39,12 @@ class StripePaymentProvider @Inject constructor(
         }
     }
 
-    override suspend fun requestPayment(item: Item): Boolean {
+    override suspend fun requestPayment(item: Item, quantity: Int): Boolean {
         if (item.id == null || item.price < 0.01) {
```

Comment:
Should you also check here that quantity is more than 0 to be sure?

Reply:
Same here

Reply:
same